mne/tests/test_report.py

```diff
@@ -83,7 +86,8 @@ def test_render_report(renderer, tmpdir):
     # Also, make sure crop range is wide enough to avoid rendering bug
     epochs.average().crop(0.1, 0.2).save(evoked_fname)
 
-    report = Report(info_fname=raw_fname_new, subjects_dir=subjects_dir)
+    report = Report(info_fname=raw_fname_new, subjects_dir=subjects_dir,
+                    projs=True)
```

Comment:
you should check that the proj appears in the rendered html. See how it's done elsewhere. I would just check that a certain string is present.

Reply:
Done in d9cc5cd